app/src/main/java/com/monstarlab/core/ui/ComposeFragment.kt

```diff
@@ -0,0 +1,24 @@
+package com.monstarlab.core.ui
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.platform.ComposeView
+import androidx.fragment.app.Fragment
+
+abstract class ComposeFragment : Fragment() {
+
+    abstract val content: @Composable () -> Unit
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return ComposeView(requireContext()).apply {
+            setContent { content.invoke() }
+        }
+    }
+}
```

Comment:
if we are using fragments we need to dispose composition when lifecycle is ded

`setViewCompositionStrategy(ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed)`

Reply:
Addressed 👍